### PR TITLE
Some changes to logging

### PR DESCRIPTION
**Pull request by tosiara**
This will change some log messages from ALR to NTC
Addresses https://github.com/Motion-Project/motion/issues/418

### CODE REVIEW

File: netcam.c [hunk 1]
```diff
@@ -2803,7 +2803,7 @@ int netcam_start(struct context *cnt)
 
     if ((retval = pthread_create(&netcam->thread_id, &handler_attribute,
                                  &netcam_handler_loop, netcam)) < 0) {
-        MOTION_LOG(ALR, TYPE_NETCAM, SHOW_ERRNO, "%s: Starting camera"
+        MOTION_LOG(NTC, TYPE_NETCAM, SHOW_ERRNO, "%s: Starting camera"
```
Comment (jogu) [inline netcam.c]: This one is on a failure path (thread creation failing) I think so perhaps should be ALR?

(Perhaps the message should mention 'error' too!)

**Comment by tosiara** [comment]
Ok, some sed and vim magic and here we go
Removed "%s:" from all the log calls and updated macro

**Comment by tosiara** [comment]
Resolved a conflict

**Comment by tosiara** [comment]
Now looks good. Requesting a review

**Comment by jogu** [comment]
This has become a little large and hard to review - can we have a pull request with just the ALR -> NTC change in it that has just one commit please? (Here's a set of git commands that might do it - I presume you have the main upstream motion project as a remote called 'golden'; change as appropriate:
```
git checkout -b alr-ntc-logging-change
git fetch golden
git reset --hard golden/master
git cherry-pick  8e03f9f8b1676769c8a94420b2bbe29818a955ff
git cherry-pick 182b1bf175f69e577d20866bbdcfb9659e81ee1d
git rebase -i golden/master
    that will launch an editor; change the 'pick' for the second commit to 's' (squash)
(be careful to watch for any conflicts here and follows the instructions git gives to resolve them before doing git rebase --continue)
git push -u origin alr-ntc-logging-change

```
then create a pull request.

**Comment by tosiara** [comment]
Sorry, didn't get. You want two separate pull requests? One for ntc-alr change and another for logging macro update?

**Comment by jogu** [comment]
That would be ideal, please - it makes it easier for me to review.

**Comment by tosiara** [comment]
Ok, I will close this PR and recreate two separate from scratch

**Pull request closed without merging**